Added: Hackathon dedicated page (#5)
UPB-RustWorkshop/upb-rustworkshop.github.io: docs/hackathon/index.md

Open-source is where **code meets community**. By contributing to Rust projects, you:

- 🛠️ **Gain real-world experience** with industry-standard tools
- 📈 **Showcase skills to employers** (GitHub = your living resume)
- 🤝 **Collaborate with global developers**
- 💡 **Learn faster** through code reviews and mentorship

**The stats don’t lie:**

- 84% of hiring managers value open-source contributions _(2023 Open Source Jobs Report)_

---

## Chose a Project to Contribute

We’ve compiled an extensive list of open-source Rust projects, ranging from CLI applications and operating systems to games and servers.

**Projects List:** **[Visit](https://docs.google.com/spreadsheets/d/17BxGBBtn4a0G2yWU_ngfr20-PePO-kfmuUku4ADwE7E/edit?usp=sharing)**

### Step 1: Choose a Project

Select a project that interests you and look for issues labeled with `good first issue`, `easy`, or any other label that indicates it’s beginner-friendly.

### Step 2: Review CONTRIBUTE.md

Read the `CONTRIBUTE.md` file carefully. This document outlines the guidelines you need to follow to ensure your contributions are eligible for integration.

### Step 3: Claim Your Issue

> To avoid multiple people working on the same issue, add a link to the issue you are working on in front of your name in the below document. This helps maintain clarity and coordination.

Reviewer: Cred ca e mai clar asa ca e vorba doar despre o organizare interna a hackatonului si nu despre un real life rule. :)

```suggestion
To avoid multiple hackaton participants working on the same issue, add a link to the issue you are working on in front of your name in the below document. This helps maintain clarity and coordination.
```

**Checklist:** **[Checklist](https://docs.google.com/spreadsheets/d/17BxGBBtn4a0G2yWU_ngfr20-PePO-kfmuUku4ADwE7E/edit?usp=sharing)**

---

> This version is clearer, more concise, and better structured. Let me know if you need further adjustments!

Reviewer: huhups

```suggestion
```

## **Your First Pull Request: A 5-Step Blueprint** 🔧

### **Step 1: Fork & Clone**

```bash
# Clone YOUR fork (replace with your fork URL)
git clone https://github.com/YOUR-USERNAME/PROJECT-NAME.git
cd PROJECT-NAME
```

### **Step 2: Create a Dedicated Branch**

```bash

> git checkout -b my-contribution  # Branch names like "fix-readme-typo" work too!

Reviewer: Cred ca ar fi util sa explicam ca numele branchului trebuie sa reprezinte concis modificarile aduse de tine. Mi-e ca asa o sa inteleaga ca trebuie sa isi numeasca branchul "my-contribution" :)

```

### **Step 3: Make Meaningful Changes**

**Examples for Rust projects:**

- Fix docs in `src/lib.rs`
- Add tests in `tests/module.rs`

> - Improve error messages in `src/error.rs`

Reviewer: Nu inteleg la ce te referi aici

Author: Exemple de ce ar putea cere un issue, nu e cel mai concis exemplu, but yeah

### **Step 4: Commit Like a Pro**

```bash
git add .
git commit -m "docs: Improve API documentation clarity"

> # Use conventional prefixes: feat, fix, docs, test, chore

Reviewer: Adauga explicatii pentru ce inseamna fix,feat, etc.

```

### **Step 5: Push & Create PR**

```bash

> git push --set-upstream origin my-contribution

Reviewer: ```suggestion
git push --set-upstream origin <YOUR-BRANCH-NAME>
```
